fvtest/threadtest/createTest.cpp

```diff
@@ -492,12 +492,12 @@ getOsStacksize(size_t stacksize)
 #if defined(LINUX) || defined(OSX)
 	/* Linux allocates 2MB if you ask for a stack smaller than STACK_MIN */
 	{
-		size_t pageSafeMinimumStack = 2 * sysconf(_SC_PAGESIZE);
+		long pageSafeMinimumStack = 2 * sysconf(_SC_PAGESIZE);
```

Comment:
https://man7.org/linux/man-pages/man3/sysconf.3.html: `sysconf` returns `long`. With `long`, you may still see an overflow for `2 * long`. `int64_t` seems large enough to handle `2 * long`.
```suggestion
		int64_t pageSafeMinimumStack = 2 * sysconf(_SC_PAGESIZE);
```